src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -440,6 +440,12 @@ public abstract partial class JsonConverter<T> : System.Text.Json.Serialization.
         public virtual void Write(System.Text.Json.Utf8JsonWriter writer, T value, System.Text.Json.JsonEncodedText propertyName, System.Text.Json.JsonSerializerOptions options) { }
         public abstract void Write(System.Text.Json.Utf8JsonWriter writer, T value, System.Text.Json.JsonSerializerOptions options);
     }
+    public sealed class JsonStringEnumConverter : System.Text.Json.Serialization.JsonConverterFactory
```

Comment:
Just for the record, the alternative here is to expose a single enum converter with a `bool` "TreatAsString" property. A new custom attribute deriving from `[JsonConverter]` would also need to be created in order to pass that `bool` on.